mllib/src/test/scala/org/apache/spark/ml/tuning/CrossValidatorSuite.scala

```diff
@@ -120,6 +120,33 @@ class CrossValidatorSuite
     }
   }
 
+  test("cross validation with parallel evaluation") {
+    val lr = new LogisticRegression
+    val lrParamMaps = new ParamGridBuilder()
+      .addGrid(lr.regParam, Array(0.001, 1000.0))
+      .addGrid(lr.maxIter, Array(0, 3))
+      .build()
+    val eval = new BinaryClassificationEvaluator
+    val cv = new CrossValidator()
+      .setEstimator(lr)
+      .setEstimatorParamMaps(lrParamMaps)
+      .setEvaluator(eval)
+      .setNumFolds(2)
+      .setParallelism(1)
```

Comment:
add `.setSeed(XXX)`

Reply:
So the seed param here is fixed by default and doesn't need to be set to ensure consistent results.  I think that's why it's not set in the other tests in this suite.  I'm not a fan of this behavior and I think it's better to explicitly set in tests, but then we should probably be consistent and set elsewhere too.  What are your thoughts on this @MLnick ?

Reply:
OK I agree.

Reply:
Yeah seed defaults to a hash of the class name. There has been debate over this (see [SPARK-16832](https://issues.apache.org/jira/browse/SPARK-16832)). Personally I also don't like that behavior, but for now that's what it is.